cogs/hl.py

```diff
@@ -58,8 +69,10 @@ async def hl(self, ctx, *, code):
 
 		if len(code) > 2000:
 			raise commands.CommandError('Code contents too long to paste.')
-
-		message = await ctx.send(code)
+		try:
+			message = await ctx.send(content=code,reference=ctx.message.reference)
```

Comment:
parameter `reference` on `discord.ext.commands.Context.send` is only available since Discord.py 1.6, but Ace is using 1.5.1 (see `requirements.txt`), so this isn't possible yet.
You would need to bump up the version of Discord.py and test if it breaks anything.